spec/controllers/admin/domain_blocks_controller_spec.rb

```diff
@@ -33,13 +39,25 @@
   end
 
   describe 'POST #create' do
-    it 'blocks the domain' do
+    it 'blocks the domain when succeeded to save' do
       allow(DomainBlockWorker).to receive(:perform_async).and_return(true)
+
       post :create, params: { domain_block: { domain: 'example.com', severity: 'silence' } }
 
       expect(DomainBlockWorker).to have_received(:perform_async)
+      expect(flash[:notice]).to eq 'Domain block is now being processed'
```

Comment:
I think my first preference would be to not even check the contents of `flash` in controller specs ... I think they are sort of an implementation detail, and if we care about the words showing up on the page, we should use feature specs.

Until we get to that point, leaving it in here is fine, but I'd prefer to check that it matches `I18n.t()` lookup instead of the string directly ... since that's what we really care that the controller is doing ... finding whatever that value is and putting it in the flash.